README.md

```diff
@@ -0,0 +1,15 @@
+# rustup-nightly
+
+A base docker image for building against the latest nightly.
```

Comment:
TIOLI you could add build instructions in this file, and link to the docker hub repo. But it's pretty straightforward of course:

```
docker build -t linkerd/rustup-nightly:latest .
```